src/App.js

```diff
@@ -1,16 +1,23 @@
-import { Routes, Route } from "react-router-dom";
-import React from "react";
+import React, { useEffect } from "react";
+import { Routes, Route, useLocation, useSearchParams } from "react-router-dom";
 
 import SideNav from "./components/side-nav/SideNav";
 import Header from "./components/header/Header";
 import Home from "./pages/home/Home";
 import Search from "./pages/search/Search";
 import Login from "./pages/login/Login";
-import { useLocation } from "react-router-dom";
 
```

Comment:
From lines 4-9, the purpose of 'import' is to allow using contents from another file and will help write modular code, i.e., splitting code into multiple files.

[Importing in React JS](https://www.educative.io/answers/what-is-importing-and-exporting-in-react-js)